package.json

```diff
@@ -78,6 +78,7 @@
   },
   "license": "MIT",
   "dependencies": {
+    "ajv": "^6.4.0",
```

Comment:
Where does ajv come into play?

Reply:
you're quick!  I was just reviewing and removing this.  Not _exactly_ sure how I ended up getting it in here.  From early efforts.